### PR TITLE
chore[elasticserach]: delete the voting config exclusion list

### PR DESCRIPTION
**Pull request by iziang**

### CODE REVIEW

File: addons/elasticsearch/templates/componentdefinition.yaml [hunk 1]
```diff
@@ -50,8 +50,30 @@ items:
             - /bin/sh
             - -c
             - |
-              echo "member $KB_LEAVE_MEMBER_POD_NAME is leaving"
-              curl --fail -v -X POST http://127.0.0.1:9200/_cluster/voting_config_exclusions?node_names=$KB_LEAVE_MEMBER_POD_NAME
+              # https://www.elastic.co/guide/en/elasticsearch/reference/7.7/add-elasticsearch-nodes.html
+              set -x
+              endpoint=http://127.0.0.1:9200
+              echo "removing node $KB_LEAVE_MEMBER_POD_NAME"
+              version=`curl --fail -v -s ${endpoint} | jq -r .version.number`
```
Comment (nayutah) [inline addons/elasticsearch/templates/componentdefinition.yaml:57]: curl 加上超时设置和 retry count 吧

File: addons/elasticsearch/templates/componentdefinition.yaml [hunk 2]
```diff
@@ -50,8 +50,30 @@ items:
...
+                echo "failed to get es version"
+                exit 1
+              fi
+              version=${version%.*}
+              if awk "BEGIN {exit !($version < 7.8)}"; then
+                url=${endpoint}/_cluster/voting_config_exclusions/$KB_LEAVE_MEMBER_POD_NAME
+              else
+                url=${endpoint}/_cluster/voting_config_exclusions?node_names=$KB_LEAVE_MEMBER_POD_NAME
+              fi
+              curl --fail -v -X POST $url
```
Comment (nayutah) [inline addons/elasticsearch/templates/componentdefinition.yaml:68]: ditto

File: addons/elasticsearch/templates/componentdefinition.yaml [hunk 3]
```diff
@@ -50,8 +50,30 @@ items:
...
+              if [ $? != 0 ]; then
+                echo "failed to add node $KB_LEAVE_MEMBER_POD_NAME to voting config exclusion list"
+                echo "may be the voting config exclusion list is full, try to remove it first"
+                curl --fail -X DELETE "${endpoint}/_cluster/voting_config_exclusions?pretty&wait_for_removal=false"
```
Comment (nayutah) [inline addons/elasticsearch/templates/componentdefinition.yaml:72]: ditto

File: addons/elasticsearch/templates/componentdefinition.yaml (hunk 1 above)
Comment (iziang) [reply addons/elasticsearch/templates/componentdefinition.yaml]: fixed

File: addons/elasticsearch/templates/componentdefinition.yaml (hunk 2 above)
Comment (iziang) [reply addons/elasticsearch/templates/componentdefinition.yaml]: fixed

File: addons/elasticsearch/templates/componentdefinition.yaml (hunk 3 above)
Comment (iziang) [reply addons/elasticsearch/templates/componentdefinition.yaml]: fixed

**Pull request merged**